### DISCUSSION TITLE
DT mod customization suggestion (bpm input)

### DISCUSSION THREAD

**Discussion by nozoz** [ppy/osu#18974] [Ideas] [answered]
I think adding an input field below the Speed Increase bar in the DT mod customization section would be a good addition to the game.
By doing this, players can find their desired bpm much faster instead of going back and forth between checking the bpm of the song and adjusting the slider.

Here's a concept picture of the idea. Please excuse my poor editing skills lol
![concept](https://user-images.githubusercontent.com/28154531/176965402-d0a6d262-920d-40dc-bb62-7cbb4df6fa21.png)

**Comment by peppy** [comment, marked as answer]
I don't know there will be a BPM entry box like you suggest, but we do plan on showing the adjusted BPM (and other statistics) somewhere visible on the mod select screen you can aim for roughly the value you want.

**Reply by smoogipoo** [reply]
It would have to show BPM plus have an option to not change gameplay variables (AR + OD + HP drain). Since to increase BPM you have to increase audio speed + reduce AR proportionally.
